eng/versioning/version_client.txt

```diff
@@ -107,8 +107,8 @@ com.azure:azure-messaging-servicebus;7.5.2;7.6.0-beta.1
 com.azure:azure-messaging-servicebus-track1-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-messaging-servicebus-track2-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-messaging-webpubsub;1.0.2;1.1.0-beta.1
-com.azure:azure-mixedreality-authentication;1.1.4;1.2.0-beta.1
-com.azure:azure-mixedreality-remoterendering;1.1.3;1.2.0-beta.1
+com.azure:azure-mixedreality-authentication;1.1.5;1.1.5
```

Comment:
I think we should first release mixed reality auth and then bump its version on main via automated PR.
 Then release mixed reality remote rendering.

Reply:
<Discussed offline> and released the patches from a release branch. I am going to close this PR since we would not be merging the source changes into main.